commands/utils/quick-pick-azure.ts

```diff
@@ -0,0 +1,66 @@
+import { ContainerRegistryManagementClient } from 'azure-arm-containerregistry';
+import { Registry } from 'azure-arm-containerregistry/lib/models';
+import * as vscode from "vscode";
+import * as azureUtils from '../../explorer/utils/azureUtils';
+import { AzureAccount, AzureSession } from '../../typings/azure-account.api';
+import * as acrTools from '../../utils/Azure/acrTools';
+import { AzureImage } from "../../utils/Azure/models/image";
+import { Repository } from "../../utils/Azure/models/Repository";
+import { AzureCredentialsManager } from '../../utils/azureCredentialsManager';
+
+/**
+ * function to allow user to pick a desired image for use
+ * @param repository the repository to look in
+ * @returns an AzureImage object (see azureUtils.ts)
+ */
+export async function quickPickACRImage(repository: Repository): Promise<AzureImage> {
+    const repoImages: AzureImage[] = await acrTools.getImages(repository);
+    console.log(repoImages);
+    let imageList: string[] = [];
+    for (let tempImage of repoImages) {
+        imageList.push(tempImage.tag);
+    }
+    let desiredImage = await vscode.window.showQuickPick(imageList, { 'canPickMany': false, 'placeHolder': 'Choose the image you want to delete' });
+    if (desiredImage === undefined) { return; }
```

Comment:
>(desiredImage === undefined) [](start = 7, length = 28)

I think a simple way to do it is `if (desiredImage)`, it will also handle `null` case